From cdf2f2e9ca41a58d49aa5206e484ab95a201dc96 Mon Sep 17 00:00:00 2001
From: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
Date: Fri, 3 Nov 2023 20:05:05 +0200
Subject: [PATCH 32/40] arm64: dts: rockchip: Enable HDMI0 PHY PLL on RK3588

This is necessary for proper handling of various display modes.

Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
---
 arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
index 39df9bfbb..dacf6a4d8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
@@ -193,6 +193,11 @@ &cpu_l3 {
 	mem-supply = <&vdd_cpu_lit_mem_s0>;
 };
 
+&display_subsystem {
+	clocks = <&hdptxphy_hdmi_clk0>;
+	clock-names = "hdmi0_phy_pll";
+};
+
 &i2c0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c0m2_xfer>;
@@ -933,6 +938,10 @@ &hdptxphy_hdmi0 {
 	status = "okay";
 };
 
+&hdptxphy_hdmi_clk0 {
+	status = "okay";
+};
+
 &vop_mmu {
 	status = "okay";
 };
-- 
2.43.0

